lib/matplotlib/backends/backend_svg.py

```diff
@@ -125,17 +208,29 @@ def __flush(self, indent=True):
             self.__write(escape_cdata(data))
             self.__data = []
 
-    ## Opens a new element.  Attributes can be given as keyword
-    # arguments, or as a string/string dictionary. The method returns
-    # an opaque identifier that can be passed to the <b>close</b>
-    # method, to close all open elements up to and including this one.
-    #
-    # @param tag Element tag.
-    # @param attrib Attribute dictionary.  Alternatively, attributes
-    #    can be given as keyword arguments.
-    # @return An element identifier.
-
     def start(self, tag, attrib={}, **extra):
+        """Opens a new element.  Attributes can be given as keyword
```

Comment:
The first line of docstrings should be short (and only one line); additional information can go in the remaining paragraphs. This should be something like:
```python
"""
Opens a new element.

Attributes can be given as keyword arguments, or ...
"""
```
This comment applies to all the docstrings.